R/bundle.R

```diff
@@ -338,6 +338,34 @@ addPackratSnapshot <- function(bundleDir, appMode) {
   if (file.exists(tempDependencyFile)) {
     unlink(tempDependencyFile)
   }
+
+  # Copy all the DESCRIPTION files we're relying on into packrat/desc.
+  # That directory will contain one file for each package, e.g.
+  # packrat/desc/shiny will be the shiny package's DESCRIPTION.
+  #
+  # The server will use this to calculate package hashes. We don't want
+  # to rely on hashes calculated by our version of packrat, because the
+  # server may be running a different version.
+  lockFilePath <- file.path(bundleDir, "packrat", "packrat.lock")
+  descDir <- file.path(bundleDir, "packrat", "desc")
+  tryCatch({
+    dir.create(descDir)
```

Comment:
Confirmed that bundle directory gets deleted. https://github.com/rstudio/rsconnect/blob/a00776881338e1e1d168eb7730efca9fe766c003/R/bundle.R#L36